<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquiposForm</class>
 <widget class="QWidget" name="EquiposForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelCodigo"><property name="text"><string>Código</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="inputCodigo"/></item>
     <item row="1" column="0"><widget class="QLabel" name="labelAnio"><property name="text"><string>Año</string></property></widget></item>
     <item row="1" column="1"><widget class="QLineEdit" name="inputAnio"/></item>
     <item row="2" column="0"><widget class="QLabel" name="labelModelo"><property name="text"><string>Modelo</string></property></widget></item>
     <item row="2" column="1"><widget class="QLineEdit" name="inputModelo"/></item>
     <item row="3" column="0"><widget class="QLabel" name="labelMarca"><property name="text"><string>Marca</string></property></widget></item>
     <item row="3" column="1"><widget class="QLineEdit" name="inputMarca"/></item>
     <item row="4" column="0"><widget class="QLabel" name="labelEstado"><property name="text"><string>Estado</string></property></widget></item>
     <item row="4" column="1">
      <widget class="QComboBox" name="inputEstado">
       <item><property name="text"><string>En arriendo</string></property></item>
       <item><property name="text"><string>En taller</string></property></item>
       <item><property name="text"><string>En reparacion</string></property></item>
      </widget>
     </item>
     <item row="5" column="0"><widget class="QLabel" name="labelBuscar"><property name="text"><string>Buscar</string></property></widget></item>
     <item row="5" column="1"><widget class="QLineEdit" name="inputBuscar"/></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botonesLayout">
     <item><widget class="QPushButton" name="btnAgregar"><property name="text"><string>Agregar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEditar"><property name="text"><string>Editar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEliminar"><property name="text"><string>Eliminar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnBuscar"><property name="text"><string>Buscar</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaEquipos"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
